src/1.21.8/main/kotlin/helpers/McClient.kt

```diff
@@ -75,10 +76,7 @@ actual object McClient {
                 .sortedBy { -it.value() }
                 .map {
                     val team = scoreboard.getPlayersTeam(it.owner())
-                    Component.empty().also { main ->
-                        team?.playerPrefix?.apply { siblings.forEach { sibling -> main.append(sibling) } }
-                        team?.playerSuffix?.apply { siblings.forEach { sibling -> main.append(sibling) } }
-                    }
+                    PlayerTeam.formatNameForTeam(team, it.ownerName())
```

Comment:
```suggestion
                    if (team == null) {
                    	it.ownerName().copy()
                    } else {
                    	Component.empty().also { main ->
	                        team?.playerPrefix?.apply { siblings.forEach { sibling -> main.append(sibling) } }
	                        team?.playerSuffix?.apply { siblings.forEach { sibling -> main.append(sibling) } }
	                    }
                    }
```

so that it produces the same result as before but still supports the other way of handling scoreboards